lib/plug/parsers.ex

```diff
@@ -193,7 +193,8 @@ defmodule Plug.Parsers do
       {"content-type", ct} ->
         case Conn.Utils.content_type(ct) do
           {:ok, type, subtype, headers} ->
-            reduce(conn, Keyword.fetch!(opts, :parsers), type, subtype, headers, opts)
+            content_type = %{type: type, subtype: subtype, headers: headers}
```

Comment:
@lexmag Should I change the name [here](https://github.com/elixir-lang/plug/pull/514/files/8a8cb8604b59fca29837db3f1d78831ed87efcc7#diff-53fad5343d1b546bc6eb0b6588f3a4a5L159) too?